Makefile

```diff
@@ -32,7 +32,7 @@ endif
 BUNDLE_METADATA_OPTS ?= $(BUNDLE_CHANNELS) $(BUNDLE_DEFAULT_CHANNEL)
 
 # Image URL to use all building/pushing image targets
-REGISTRY ?= quay.io/rhibmcollab
+REGISTRY ?= quay.io/open-cluster-management
```

Comment:
Should we use open-cluster-management repo? For local development I usually use my personal quay repo. Last I recall, I dont think we have push access w/ our default accounts to open-cluster-management repo.

Reply:
I think we aren't technically supposed to be using rhibmcollab and wanted to match it with where our images are officially stored, with the expectation that anyone who wants to build our repo will push to their own registry. @nathanweatherly WDYT?

Reply:
i wonder if we could get a squad-wide org for our test images-- that way we could have a non-prod org that we use for testing (so we won't accidentally override some image built in the pipeline), but it's still not an "ibm" org

Reply:
I've reverted the name change